masterlist.yaml

```diff
@@ -1975,6 +1975,9 @@ plugins:
         util: SSEEdit v3.2.1
   - name: 'Ars Metallica.esp'
     url: ['https://www.nexusmods.com/skyrimspecialedition/mods/321/']
+    after:
+      - 'TheWulfPanda - Faction Teleport Spells.esp'
+      - 'TheWulfPanda - Faction Teleport Spells Less Magicka Increased Cast Time.esp'
```

Comment:
These load after rules are not needed as its only a weather region edit that is being overwritten,
 which isn't critical to `Ars Metallica.esp`

![image](https://user-images.githubusercontent.com/1944639/42129230-d0764a7e-7cb6-11e8-84cd-631df1d32468.png)

Reply:
Okay, I rebased my PR.